File: login_school_network.py
import re, sys

from requests import post, get

def login():
    url = "http://222.199.222.14"
    data = {
            "DDDDD": "你的学号",
            "upass": "密码",
            "v6ip": "你的ipv6地址",
            "0MKKey": "一个不知道叫啥的字串"
            }
    post(url, data=data).content.decode("gbk")
    return

def logout():
    url = "http://222.199.222.14/F.htm"
    get(url).content.decode("gbk")
    return

if __name__ == "__main__":
    if(len(sys.argv) > 2):
        print("参数过多，请确认参数输入正确")
    elif(len(sys.argv < 2)):
        print("参数太少啦，你总得输一个吧")
    elif(sys.argv[1] == "--login"):
        login()
        print("成功登陆校园网")
    elif(sys.argv[1] == "--logout"):
        logout()
        print("成功注销校园网")
    else:
        print("请求失败，请检查脚本参数是否输入正确")

File: login_school_network.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
	"net/url"
	"os"
)

func main() {
	if len(os.Args) > 2 {
		fmt.Println("参数过多，请确认输入参数正确！")
	} else if len(os.Args) < 2 {
		fmt.Println("参数太少啦，你至少得输入一个吧！")
	} else if os.Args[1] == "--login" {
		login()
	} else if os.Args[1] == "--logout" {
		logout()
	} else {
		fmt.Println("请求失败，请检查脚本参数是否正确")
	}
	return
}

func login() {
	resp, err := http.PostForm("http://222.199.222.14", url.Values{"DDDDD": {"你的学号"}, "upass": {"密码"}, "v6ip": {"你的ipv6地址"}, "0MKKey": {"不知道是啥的字串"}})
	if err == nil {
		fmt.Println("成功登陆校园网")
	}
	defer resp.Body.Close()
	body, err := ioutil.ReadAll(resp.Body)
	if err != nil {
		fmt.Printf("%s", body)
	}
	return
}

func logout() {
	resp, err := http.Get("http://222.199.222.14/F.htm")
	if err == nil {
		fmt.Println("成功注销校园网")
	}
	defer resp.Body.Close()
	body, err := ioutil.ReadAll(resp.Body)
	if err != nil {
		fmt.Printf("%s", body)
	}
	return
}
